lib/utapi/utilities.js

```diff
@@ -38,7 +38,6 @@ function _listBucketMetrics(host,
             const responseBody = JSON.parse(body.join(''));
             if (response.statusCode >= 200 && response.statusCode < 300) {
                 // eslint-disable-next-line no-console
-                console.log(responseBody);
```

Comment:
Technically, a `process.stdout.write()` makes more sense outside of a browser, and would lift the need to disable the linter. But that entirely orthogonal to this PR anyway.